main.go

```diff
@@ -14,6 +14,11 @@ import (
 )
 
 var (
+	coin = kingpin.Flag(
```

Comment:
Let's make this an [Enum](https://godoc.org/github.com/alecthomas/kingpin#ArgClause.Enum) to make sure the input is validated. 

I'd also like to keep this backwards-compatible, so let's preserve the existing behavior by having this default to "BTC" if the flag is unspecified.

Reply:
done